src/data/cassandra/impl/Result.hpp

```diff
@@ -99,6 +101,14 @@ extractColumn(CassRow const* row, std::size_t idx)
         auto const rc = cass_value_get_int64(cass_row_get_column(row, idx), &out);
         throwErrorIfNeeded(rc, "Extract int64");
         output = static_cast<DecayedType>(out);
+    } else if constexpr (std::is_convertible_v<DecayedType, boost::uuids::uuid>) {
+        CassUuid uuid;
+        auto const rc = cass_value_get_uuid(cass_row_get_column(row, idx), &uuid);
+        throwErrorIfNeeded(rc, "Extract uuid");
+        std::string uuidStr(CASS_UUID_STRING_LENGTH, '0');
+        cass_uuid_string(uuid, uuidStr.data());
+        uuidStr.pop_back();  // remove the last \0 character
```

Comment:
Is there really no cleaner way to do this?

Reply:
`CASS_UUID_STRING_LENGTH` is 37 sybmols: 36 for uuid and one for `\0`. I think `cass_uuid_string()` expects a buffer with the length `CASS_UUID_STRING_LENGTH`. So probably there is no better way.